src/main/java/frc/robot/Constants.java

```diff
@@ -19,9 +19,12 @@
     //controller port
     public static final int controllerport = 0;
     //motor controller ports
-    public static final int outputMotorPort = 2;
```

Comment:
This should still be where the output motor port is. There is no reason to move it.